Description: Ensure that parameters to zkCli.sh are correctly
 passed to the supporting Java code.
Author: John Eikenberry <jae@zhar.net>
Bug: https://issues.apache.org/jira/browse/ZOOKEEPER-1403

Index: zookeeper/bin/zkCli.sh
===================================================================
--- zookeeper.orig/bin/zkCli.sh	2012-01-03 10:21:37.000000000 +0000
+++ zookeeper/bin/zkCli.sh	2012-03-02 11:13:34.076917800 +0000
@@ -38,4 +38,4 @@
 
 $JAVA "-Dzookeeper.log.dir=${ZOO_LOG_DIR}" "-Dzookeeper.root.logger=${ZOO_LOG4J_PROP}" \
      -cp "$CLASSPATH" $CLIENT_JVMFLAGS $JVMFLAGS \
-     org.apache.zookeeper.ZooKeeperMain $@
+     org.apache.zookeeper.ZooKeeperMain "$@"
